core/index.ts

```diff
@@ -21,10 +21,14 @@ import UserProfile from './lib/UserProfile';
 import IExecJsPathResult from '@secret-agent/injected-scripts/interfaces/IExecJsPathResult';
 import { IRequestInit } from 'awaited-dom/base/interfaces/official';
 import IAttachedState from '@secret-agent/injected-scripts/interfaces/IAttachedStateCopy';
-import Signals = NodeJS.Signals;
 import Log from '../commons/Logger';
+import { createReplayServer } from '@secret-agent/session-state/api';
+import ISessionReplayServer from '../session-state/interfaces/ISessionReplayServer';
+import Signals = NodeJS.Signals;
 
 const { log } = Log(module);
+const shouldStartReplayServer = Boolean(JSON.parse(process.env.SA_SHOW_REPLAY ?? 'true'));
```

Comment:
In this case, we're parsing the env var at the top of the file, which seems to make a lot more sense, even if it's only used once.